src/main/java/frc/robot/hardware/rev/motors/BrushlessSparkMAXMotor.java

```diff
@@ -13,14 +14,23 @@ public class BrushlessSparkMAXMotor extends SparkMaxMotor implements Controllabl
 
 	private final SysIdCalibrator.SysIdConfigInfo sysIdConfigInfo;
 
-	public BrushlessSparkMAXMotor(String logPath, SparkMaxWrapper motor, SysIdRoutine.Config sysidConfig) {
-		super(logPath, motor);
+	public BrushlessSparkMAXMotor(
+		String logPath,
+		SparkMaxWrapper motor,
+		MechanismSimulation mechanismSimulation,
+		SysIdRoutine.Config sysidConfig
+	) {
+		super(logPath, motor, mechanismSimulation);
 		if (motor.getMotorType() != SparkLowLevel.MotorType.kBrushless) {
 			throw new IllegalArgumentException("inserted BrushedSparkMAXMotor to BrushlessSparkMAXMotor!");
 		}
 		this.sysIdConfigInfo = new SysIdCalibrator.SysIdConfigInfo(sysidConfig, false);
 	}
 
+	public BrushlessSparkMAXMotor(String logPath, SparkMaxWrapper motor, SysIdRoutine.Config sysidConfig) {
+		this(logPath, motor, null, sysidConfig);
```

Comment:
are you sure null is the correct thing to insert? is there no default constructor?

Reply:
Same as the other comment